tests/test_bibliotheca.py

```diff
@@ -630,6 +630,11 @@ def test_circulation_sweep_discovers_work(self):
         )
         monitor.process_items([identifier])
 
+        # Validate that the HTTP request went to the /items endpoint.
+        request = self.api.requests.pop()
+        url = request[1]
+        eq_(url, self.api.full_url("items") + "/" + identifier.identifier)
+
```

Comment:
These tests are better because they eliminate the possibility that we are still going to the /circulation/items endpoint instead of the /items endpoint.